content_macros/event_fields update css and add a slot for extra fields

cillianderoiste opened the pull request:
Resync the event view with the prototype. Move the description into the body and add an extra slot into the event_fields macro to allow additional fields to be included when customized (which we need for iKath).

cillianderoiste committed:
content_macros/event_fields update css and add a slot for extra fields

ghost commented:
Refer to this link for build results (access rights to CI server needed): 
http://jenkins.ploneintranet.net/job/Plone.Intranet.Pull.Request.Builder/220/
Test PASSed.

cillianderoiste committed:
event_fields: include the Description field in the main content area

ghost commented:
Refer to this link for build results (access rights to CI server needed): 
http://jenkins.ploneintranet.net/job/Plone.Intranet.Pull.Request.Builder/223/
Test PASSed.

mattss commented on src/ploneintranet/workspace/browser/templates/content_macros.pt:
```diff
@@ -53,8 +53,17 @@
+      <fieldset class="vertical fancy">
+        <label class="description">
+          Description
+          <textarea name="description" placeholder="Enter a description of the task"
```
Should this read 'description of the event'?

cillianderoiste replied:
Fixed, thanks!

cillianderoiste committed:
event_view: remove duplicate Description field / hide metadata

In the prototype there is a description field in both the body and the
collapsible "metadata" field. We can't submit two fields with the same
name (or we end up with a list of strings). Since there are no metadata
fields in the collapsible menu, I've commented it out altogether.

cillianderoiste committed:
event_fields: fix copy-and-paste fail: "Task" -> "Event"

ghost commented:
Refer to this link for build results (access rights to CI server needed): 
http://jenkins.ploneintranet.net/job/Plone.Intranet.Pull.Request.Builder/224/
Test PASSed.

ghost commented:
Refer to this link for build results (access rights to CI server needed): 
http://jenkins.ploneintranet.net/job/Plone.Intranet.Pull.Request.Builder/225/
Test PASSed.
